openreview/venue_request/venue_request.py

```diff
@@ -185,6 +185,76 @@ def setup_review_stage(self):
             }
         ))
 
+    def setup_comment_stage(self):
+
+        comment_stage_content = {
+            'commentary_start_date': {
+                'description': 'When does official and/or public commentary begin? Please use the following format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59)',
+                'value-regex': r'^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$',
+                'order': 27
+            },
+            'commentary_end_date': {
+                'description': 'When does official and/or public commentary end? Please use the following format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59)',
+                'value-regex': r'^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$',
+                'order': 28
+            },
+            'allow_unsubmitted_reviewers_to_comment': {
```

Comment:
merge these two fields in a single one:

participants:

- Program Chairs
- All Area Chairs
- Paper Area Chairs
- All Reviewers
- Paper Reviewers
- Submitted Reviewers
- Authors

Reply:
this means that the selected values will be invitees and part of the readers selection.